core/src/main/java/com/linecorp/armeria/server/PathMappingContext.java

```diff
@@ -68,12 +69,11 @@
     MediaType consumeType();
 
     /**
-     * Returns a list of {@link MediaType} that is ordered by client-side preferences.
-     * It only includes the media types supported by the virtual host. {@code null} would be returned
-     * if the virtual host does not support media type negotiation.
+     * Returns a list of {@link MediaType}s that are specified in {@link HttpHeaderNames#ACCEPT} in the order
+     * of client-side preferences. If the client does not send the header, this will contain only
+     * {@link MediaType#ANY_TYPE}.
      */
-    @Nullable
-    List<MediaType> produceTypes();
+    List<MediaType> acceptTypes();
```

Comment:
I think that it's good try to change this name from a `Request`'s point of view. Don't we need to consider changing `consumeType` to something like `contentType` as well?

Reply:
That's a good suggestion. I was sure that `acceptTypes` is better naming because I changed the logic which purely extracts the content types in the `Accept` header but was not sure that I could change `consumeType`. Let me rename it. Thanks!